### ISSUE TITLE
macOS support

### ISSUE THREAD

**Issue by ghost** [revng/orchestra#145]
Are there any plans to support running on MacOS？

**Comment by aleclearmind** [comment]
Yes, but that will require a while. We're thinking about adopting `nix` as a build system, which will significantly facilitate that.

Consider however that the UI can run from any browser, so if you can have access to a Linux server, you can still use rev.ng from remote.
